components/ui/top-tab-navigation/index.tsx

```
@@ -71,82 +105,150 @@ const TopTabNavigation: React.FC<TopTabNavigationProps> = ({
+                alignSelf: "stretch",
+                justifyContent: "flex-start",
+                paddingTop: 16,
+                minHeight: Dimensions.get("window").height * 0.8,
+                paddingHorizontal: 16,
+                opacity: collapseAnim,
+              },
+            ]}
+          >
+            {tabs.map((tab, index) =>
+              tab.title == "---" ? (
+                <Divider />
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)